shellcheck.sh

```diff
@@ -1,22 +1,11 @@
 #!/usr/bin/env bash
-
 set -e
 
-targets=()
-while IFS= read -r -d $'\0'; do
-  targets+=("$REPLY")
-done < <(
-  find . -type f \( -name \*.bash -o -name \*.sh \) -print0
-  find . -type f -name '*.bats' -not -name '*_no_shellcheck*' -print0
-  find libexec -type f -print0
```

Comment:
Dir you compare the -- list output before and after? I think your change will omit all das les in libexec.